internal/version/version.go

```diff
@@ -1,3 +1,31 @@
 package version
 
+import (
+	"runtime/debug"
+)
+
 const Version = "0.3.0"
+
+// GetVersion returns
+// the version number
+func GetVersion() string {
+	return Version
+}
+
+// GetShortRevision returns a short
+// commit or checkout revision
+// from build information
+func GetShortRevision() string {
+	if info, ok := debug.ReadBuildInfo(); ok {
+		for _, setting := range info.Settings {
+			if setting.Key == "vcs.revision" {
+				var shortRevision string
+				if len(setting.Value) > 7 {
+					shortRevision = setting.Value[:7]
```

Comment:
It looks like this will return an empty string if the value length is >= 7. Do we want to do something like this to be safer and always return the value?

```suggestion
				shortRevision := setting.Value
				if len(shortRevision) > 7 {
					shortRevision = setting.Value[:7]
```

Reply:
Good catch!